worker/src/unstract/worker/worker.py

```diff
@@ -112,40 +116,41 @@ def stream_logs(
     ) -> None:
         for line in container.logs(stream=True, follow=True):
             log_message = line.decode().strip()
-            logger.info(f"Log message: {log_message}")
+            self.logger.info(f"[{self.container_name}] - {log_message}")
             self.process_log_message(
                 log_message=log_message,
                 tool_instance_id=tool_instance_id,
                 channel=channel,
             )
 
-    def get_valid_log_message(
-        self, log_message: str, channel: Optional[str] = None
-    ) -> Optional[dict[str, Any]]:
+    def get_valid_log_message(self, log_message: str) -> Optional[dict[str, Any]]:
+        """Get a valid log message from the log message.
+
+        Args:
+            log_message (str): str
+
+        Returns:
+            Optional[dict[str, Any]]: json message
+        """
         try:
             log_dict = json.loads(log_message)
-            if not isinstance(log_dict, dict):
-                logger.error(f"Received invalid log: {log_dict}")
-            else:
+            if isinstance(log_dict, dict):
```

Comment:
Do you think this check is necessary @muhammad-ali-e ?

Reply:
This to confirm the return object is a dict. We exepecting it for FE logs to publish to pubsub